recipe/meta.yaml

```diff
@@ -9,22 +9,29 @@ source:
   - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/tf_keras-{{ version }}.tar.gz
     sha256: fda97c18da30da0f72a5a7e80f3eee343b09f4c206dad6c57c944fb2cd18560e
   - fn: LICENSE
-    url: https://raw.githubusercontent.com/keras-team/tf-keras/master/LICENSE
+    url: https://raw.githubusercontent.com/keras-team/tf-keras/v{{ version }}/LICENSE
     sha256: 58d1e17ffe5109a7ae296caafcadfdbe6a7d176f0bc4ab01e12a689b0499d8bd
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  # skip win because of missing tf>2.10
+  skip: True  # [win or s390x]
+  # skip osx-64 because of failed import of tf-keras:
+  # ImportError: cannot import name 'training_distributed_v1' from 'tensorflow.python.keras.engine'
+  skip: True  # [osx and x86_64]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
```

Comment:
This may be a silly question but should this number be incremented rather than decremented? It looks like a rebuild of the same version, if I'm not mistaken.